numpy/lib/_arraypad_impl.py

```diff
@@ -848,7 +848,7 @@ def pad(array, pad_width, mode='constant', **kwargs):
 
     elif mode in stat_functions:
         func = stat_functions[mode]
-        length = kwargs.get("stat_length", None)
+        length = kwargs.get("stat_length")
```

Comment:
It could be argued that the original was better because it doesn't require the reader to remember the default.

Reply:
The `.get()` function is quite basic. If the reader doesn't know that it returns `None` by default, then he probably doesn't know what the function is about and would have to look it into the documentation any way.